migrations/1665585926315-add-default-theme-to-user.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { runMigration } = require('../utils/mongodb')
+
+module.exports.up = runMigration(async (db) => {
+  await db.collection('users').updateMany(
+    { theme: { $exists: false } },
+    {
+      $set: {
+        theme: 'light',
+      },
+    }
+  )
+})
+
+module.exports.down = runMigration(async (db) => {
+  await db.collection('users').updateMany({}, { $unset: { theme: '' } })
```

Comment:
thought: This will delete any theme set, including if someone had chosen dark. However, for me the difference here from #804 is that dark mode hasn't been rolled out yet so no one could have made a preference. So I'm fine with it here. Also there is no way to distinguish someone saving light mode and having the default which is different than the other PR as well.